secure_message_v2/views/batch_requests.py

```diff
@@ -0,0 +1,29 @@
+import logging
+from typing import Optional
+
+from flask import Blueprint, current_app
+from flask_httpauth import HTTPBasicAuth
+from structlog import wrap_logger
+
+from secure_message_v2.controllers.threads import marked_for_deletion_by_closed_at_date
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+batch_request_bp = Blueprint("batch_request_bp", __name__)
+auth = HTTPBasicAuth()
+
+
+@auth.verify_password
+def verify_password(username: str, password: str) -> Optional[str]:
+    if (
+        username == current_app.config["SECURITY_USER_NAME"]
+        and password == current_app.config["SECURITY_USER_PASSWORD"]
+    ):
+        return username
+
+
+@batch_request_bp.route("/mark_thread_for_deletion", methods=["PATCH"])
```

Comment:
Couldn't make my mind up, PATCH, PUT, DELETE, it's not really a delete as it doesn't well delete anything, it's not really a patch as I would expect something in the body, it's not really a put as it doesn't put anything in body and doesn't update a full record